package/gluon-core/luasrc/lib/gluon/upgrade/300-firewall-rules

```diff
@@ -57,8 +57,16 @@ for _, zone in ipairs({'mesh', 'local_client', 'wired_mesh'}) do
 	uci:delete('firewall', zone .. '_ICMPv6_out')
 end
 
-uci:section('firewall', 'rule', 'local_client_ICMPv4_in', {
-	src = 'local_client',
+-- ToDo Remove in v2022.x
+uci:delete('firewall', 'local_client_ICMPv6_in')
+local local_client_networks = uci:get_list('firewall', 'local_client', 'network')
+if local_client_networks ~= nil then
+	uci:set_list('firewall', 'loc_client', 'network', local_client_networks)
+	uci:delete('firewall', 'local_client')
+end
```

Comment:
This part can be removed. `network` is completely rewritten on each update (see line 25 of this file), so we don't need to migrate anything.